getting_started/Run_MaxText_via_xpk.md

```diff
@@ -84,7 +84,9 @@ after which log out and log back in to the machine.
     gcloud config set project $PROJECT_ID
     bash docker_upload_runner.sh CLOUD_IMAGE_NAME=${USER}_runner
```

Comment:
@rwitten I was curious if you had a good way that we could avoid the user needing to upload the runner at all?

Do we build the dependency image already and then they could just use that as their base docker image? Or is this worth (could have mismatch to local maxtext)

Reply:
This is a good idea. XPK doesn't support this yet so I am thinking of adding this feature as a precursor to the PR here

Reply:
Actually I am incorrect. XPK does support this and I somehow didn't think it did.